library/default/project/config/base_osw_floating.yaml

```
@@ -0,0 +1,16 @@
+name: base_floating_2024
+weather: era5_41.0N_125.0W_1989_2019.csv
+service_equipment:
+  - [ctv.yaml, 3]
+  - cab.yaml
+  - dsv.yaml
+  - ahv.yaml
+layout: base_osw_floating.csv
+inflation_rate: 0
+fixed_costs: fixed_costs_osw_floating.yaml
+workday_start: 7
+workday_end: 19
+start_year: 1999
+end_year: 2019
```

```suggestion
end_year: 2018
```
Using 2019 will yield a 21-year simulation since the end_year is inclusive of the listed year.

---

See [20 years of operation instead of 21](https://github.com/WISDEM/WOMBAT/pull/216/commits/70a89f607f05d5547b70a3d7289d6175e4434efc)